public/app/features/plugins/admin/pages/Browse.tsx

```diff
@@ -53,6 +53,10 @@ export default function Browse({ route }: GrafanaRouteComponentProps): ReactElem
     history.push({ query: { filterBy: 'all', filterByType: 'all', q } });
   };
 
+  const onDisplayMode = (value: string) => {
+    history.push({ query: { displayAs: value } });
```

Comment:
nit: I'm wondering if it would be a better user experience to have this `displayAs` in the state so it's treated more like a preference that persists rather than using the url params which can be reset more easily. (e.g clicking the plugins breadcrumb link in the PluginDetails page or navigating away then back again.

Reply:
Done in https://github.com/grafana/grafana/pull/39466/commits/e78e56a71e69519c2f7fa91f7efea8dc98f704bf.